### PR TITLE
Add-email command

### PR DESCRIPTION
**Pull request by ZuoZhuolin**

### CODE REVIEW

File: src/seedu/addressbook/commands/EditEmailCommand.java [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.exception.IllegalValueException;
+import seedu.addressbook.data.person.Email;
+import seedu.addressbook.data.person.Person;
+import seedu.addressbook.data.person.UniquePersonList.PersonNotFoundException;
+
+/**
+ * Edit the email of a person in the address book.
+ */
+public class EditEmailCommand extends Command{
```
Comment (marcusngwenjian) [inline src/seedu/addressbook/commands/EditEmailCommand.java:12]: There should be a whitespace between 'Command' and '{'

File: src/seedu/addressbook/parser/Parser.java [hunk 2]
```diff
@@ -96,7 +99,10 @@ public Command parseCommand(String userInput) {
                 
             case EditPhoneCommand.COMMAND_WORD:
                 return prepareEditPhone(arguments);
-
+
```
Comment (marcusngwenjian) [inline src/seedu/addressbook/parser/Parser.java:102]: case should align to switch to conform to java coding standards

https://oss-generic.github.io/process/codingstandards/coding-standards-java.html

**Comment by marcusngwenjian** [comment]
@CS2103AUG2016-W15-C3/testing 
@CS2103AUG2016-W15-C3/integration
 Code quality has been checked. Ready for testing and integration.

**Comment by turtle96** [comment]
Branch should be renamed Add-edit-email or similar for clarity

**Comment by turtle96** [comment]
Please check and test code again:
1. Help command parameters error
2. Command renames entire list of persons' emails to the new email, should be just one person

**Pull request merged**